React/Week1/my-app-week1-task/app/Products/Product-one/page.js

```diff
@@ -0,0 +1,8 @@
+export default function ProductOnePage() {
```

Comment:
currently this is live on {{root-url}}/Products/Product-one

it is an ok route, and the page loads correctly, but there is a better and more simple way to do nested routes.

you can find a short description in the nex.js documentation here 'creating nested routes':

https://nextjs.org/docs/app/getting-started/layouts-and-pages